packages/react/src/hooks/useStorageURL.ts

```diff
@@ -14,19 +14,21 @@ export interface UseStorageURLResult {
  */
 export const useStorageURL = (
   key: string,
-  options?: S3ProviderGetConfig
+  options?: S3ProviderGetConfig,
+  fallbackURL?: string
 ): UseStorageURLResult & { fetch: () => () => void } => {
   const [result, setResult] = React.useState<UseStorageURLResult>({
     isLoading: true,
   });
 
   // Used to prevent an infinite loop on useEffect, because `options`
   // will have a different reference on every render
-  const serializedOptions = JSON.stringify(options);
+  const serializedOptions = JSON.stringify(options ?? {});
 
   const fetch = () => {
     setResult({ isLoading: true });
 
+    const options = JSON.parse(serializedOptions) as S3ProviderGetConfig;
```

Comment:
Please do not reuse namespaces already declared in the outer scope of a function. It increases the cognitive load for readers attempting to grok what is happening when two variables share a name